src/controller/CHIPDevice.cpp

```diff
@@ -55,16 +56,54 @@ using namespace chip::Callback;
 namespace chip {
 namespace Controller {
 
-CHIP_ERROR Device::SendMessage(System::PacketBufferHandle buffer, PayloadHeader & payloadHeader)
+CHIP_ERROR Device::LoadSecureSessionParametersIfNeeded(bool & didLoad)
```

Comment:
Why did this get moved?  There's no obvious reason to move it, and it makes review much more difficult (have to check it line by line for differences, etc).  Please put it back where it used to be.

Reply:
There are two SendMessage, this PR delete the first one, which cause the confusion on review, merged the second one to the first one solve this problem.